linked_list/main.py

```diff
@@ -44,16 +45,20 @@ class Node(object):
     """
 
     def __init__(self, elem, next=None):
-        pass
+        self.elem = elem
+        self.next = next
+
 
     def __str__(self):
-        pass
+        return str(self.elem)
 
     def __eq__(self, other):
-        pass
+        if self.elem == other.elem:
+            return True
+        return False
 
     def __repr__(self):
-        pass
+        return "Element: {}, next = {}".format(self.elem, self.next)
```

Comment:
While this works, I think you should change Element to Node.

While its easy to tell whats what in this case, if you were working on a more complicated project, multiple objects could all have an Element section, if you're returning the repr(object), you want to know that it's a node.